apps/librelingo_json_export/tests/test_export.py

```diff
@@ -401,20 +401,36 @@ def test_calls_define_word_the_correct_number_of_times(self, _define_word):
         _define_words_in_sentence(fakes.course1, "foo bar baz", False)
         assert _define_word.call_count == 3
 
+    @patch("librelingo_json_export.dictionary._define_word")
+    def test_calls_define_word_the_correct_number_of_times_with_spaces(self, _define_word):
+        _define_words_in_sentence(fakes.course1, "foo {bar baz}", False)
+        assert _define_word.call_count == 2
+
     @patch("librelingo_json_export.dictionary._define_word")
     def test_calls_define_word_with_the_correct_data(self, _define_word):
         is_in_target_language = fakes.fake_value()
         fake_word = str(fakes.fake_value())
         _define_words_in_sentence(fakes.course1, fake_word, is_in_target_language)
         _define_word.assert_called_with(fakes.course1, fake_word, is_in_target_language)
 
+    @patch("librelingo_json_export.dictionary._define_word")
+    def test_calls_define_word_with_the_correct_data_with_spaces(self, _define_word):
```

Comment:
```suggestion
    def test_calls_define_word_with_the_correct_word_that_has_space(self, _define_word):
```

Reply:
I am thinking you can probably merge this test case with this other one: https://github.com/kantord/LibreLingo/pull/1452/files#diff-ab9c50ea4369e0645fb5e151128d58dde9e38e4879661c5890f0be840cb5e650R404

You could basically assert the full list of calls, which would both assert that the call arguments are correct, as well as that they were called the correct number of times.

Something like this maybe? https://stackoverflow.com/a/24063759

Reply:
Here also, I copied this structure from the surrounding tests: [here](https://github.com/kantord/LibreLingo/pull/1452/files#diff-ab9c50ea4369e0645fb5e151128d58dde9e38e4879661c5890f0be840cb5e650L400) and [here](https://github.com/kantord/LibreLingo/pull/1452/files#diff-ab9c50ea4369e0645fb5e151128d58dde9e38e4879661c5890f0be840cb5e650L405).

If you'd like me to merge those two as well, I can take a crack at it.

Reply:
> If you'd like me to merge those two as well, I can take a crack at it.

sounds good to me. If you do that, I'll be very grateful.